OKR.md

```diff
@@ -2,6 +2,32 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q3
+
+**The Connectivity Magic story is figured out and solved**
+
+- A js-ipfs Daemon will always find other IPFS Daemons (js & go) in the LAN 
+- A js-ipfs node (Browser & Desktop) always finds files that are on the Gateways
+- The rendezvous protocol is a sound solution for DApps (e.g. PeerPad & others)
+- The DHT is enabled by default in js-ipfs
+- Handles correctly a scenario where the number of Connections grows (through Conn Management Strategy)
+
+**Daemon is as reliable as it can be**
+
+- A Daemon runs for 10 days without a crash
```

Comment:
In this KR, the implicit things are:

- Setting up a daemon that gets monitored should be easily reproducible
- It should be tested on our IPFS Infra
- The daemon should be exposed to the "elements" (rest of the network)